### DISCUSSION TITLE
Fitting snRNA-seq to spatial seq

### DISCUSSION THREAD

**Discussion by Jiayi-Zheng** [BayraktarLab/cell2location#334] [Q&A]
Hello, I wanna ask if anyone has tried integrating single nuclei RNA result with spatial sequencing results? I'm not sure how much difference is there exactly between the scRNA-seq and snRNA-seq and if the snRNA-seq could fit well into the spatial. I know this likely works given the strong cell type signature, but would there be performance difference between these two?

**Comment by vitkl** [comment]
Hi @Jiayi-Zheng 

Cell2location paper shows examples of using both single nuclei reference (mouse brain) and single cell reference (lymph node, gut, Zheng et al mouse brain) - so both can be used. Cell2location is more accurate than other methods especially when using single nuclei reference because of technology difference terms in the model.
